pyproject.toml

```diff
@@ -34,7 +34,7 @@ pyproj = "==3.6.1"
 python-dotenv = "==1.0.1"
 pytz = "^2024.1"
 rtreelib = "==0.2.0"
-werkzeug = "^3.0.3"
+werkzeug = "^3.0.6"
```

Comment:
Flagging a dependency update being included in this changeset - it's fine in this case, this is a development dependency, just didn't want it to go unnoticed!